server/app.js

```diff
@@ -17,6 +21,22 @@ app.use(compression({ threshold: 0 }));
 mongoose.Promise = bluebird;
 mongoose.connect(process.env.MONGO_URI);
 
+const logDirectory = path.join(__dirname, 'log');
```

Comment:
@BerkeleyTrue i don't understand, what you means by couple ? Can you please help me to understand ? And about the file you means something like this ?
app.use(morgan('combined', {stream: {
write: (str) =>  log.info(str);
}}));

Tks !